lib/Backend/Lower.cpp

```diff
@@ -2570,6 +2570,7 @@ Lowerer::LowerRange(IR::Instr *instrStart, IR::Instr *instrEnd, bool defaultDoFa
         case Js::OpCode::Throw:
         case Js::OpCode::InlineThrow:
         case Js::OpCode::EHThrow:
+            instr->m_func->SetHasThrow();
```

Comment:
It might be good to set these earlier in the JIT as other people might want to use these.  I'd set this one in the initial backwards phase, after FgBuild has eliminated the true unreachable code.